### ISSUE TITLE
Replicating results

### ISSUE THREAD

**Issue by emanuelepesce** [shariqiqbal2810/maddpg-pytorch#1]
Hi,
which setting do you use to replicate the paper results?

On simple_tag scenario the best configuration so far is:
- 128 unites 2 layers MLP for policy 
- 64 unites 1 layer MLP for critic
- 60k episodes

When I run the model, after the training I get some good cases (like those ones you showed [here](https://github.com/shariqiqbal2810/maddpg-pytorch), but even lots of cases where it doesn't work so well. Is it quite normal or I'm doing something wrong?

I still not able to replicate results for the simple_speaker_listener scenario.

**Comment by shariqiqbal2810** [comment]
Try using discrete action spaces (you can just use the `--discrete_action` flag). Despite the original paper claiming to use continuous policies, I noticed that, in the code release they were using discrete policies, and, if you look carefully, you can see that they're using discrete policies in the [video](https://www.youtube.com/watch?v=QCmBo91Wy64) that goes along with the paper. That's how I got most of the results that I put in the README, with the exception of the physical deception scenario.

**Comment by emanuelepesce** [comment]
Thank you, indeed it works pretty well with discrete actions!

However, I think I'm missing something. Correct me if I'm wrong, but the DDPG (which is the core for MADDPG) shouldn't assume continuous actions for the deterministic policy?

**Comment by shariqiqbal2810** [comment]
You're totally right. I was a bit thrown off by this too. I think the answer is that the algorithm becomes somewhat more like [stochastic value gradients](https://arxiv.org/pdf/1510.09142.pdf) due to the stochastic policy, rather than DDPG. Essentially you're calculating a one-sample monte-carlo estimate of the gradient of the value function at each step.
